scipy/interpolate/_fitpack_py.py

```diff
@@ -564,6 +564,30 @@ def spalde(x, tck):
     .. [3] Dierckx P. : Curve and surface fitting with splines, Monographs on
        Numerical Analysis, Oxford University Press, 1993.
 
+    Examples
+    --------
+    >>> from scipy.interpolate import spalde
+    >>> x = (0.5)
```

Comment:
```suggestion
    >>> x = 0.5
```

Reply:
I will modify the file with this and figure sugestion